include/cppkafka/exceptions.h

```diff
@@ -38,6 +38,18 @@
 
 namespace cppkafka {
 
+// Based on syslog.h levels
+enum LogLevel {
```

Comment:
Maybe this belongs on a new `cppkafka/logging.h` file? Not sure what else would go there but I think it's slightly off in this `exceptions.h`. Also, I'd use `enum class` so this is better scoped.

Reply:
Ok. I didn't use the class as you are not using it anywhere else, but I agree it's a better approach. Unfortunately this means that convertion enum->int has to be cast now.

Reply:
Looks like it's being used on half on the cases. This could be cleaned up in some major version.